orchestra/static/orchestra/timing/time-entry.js

```diff
@@ -96,6 +96,7 @@
           }
         }
         entry.date = moment(entry.date).startOf('day');
+        entry.assignment = data.assignment || undefined;
```

Comment:
won't it be undefined by default?  more for my edification, what are the other possiblities?